content/speaking.md

```diff
@@ -0,0 +1,17 @@
++++
+title = "Speaking"
++++
+
+We welcome speakers to submit talks to our meetups. We'll be happy to help you find a time that works for you.
+
+We are open to speakers from all backgrounds and levels of experience. You can either give a 20-30 minute presentation or participate in our Ignite/Pecha Kucha sessions. This is a short 5 minute format with some 
+
+## How to submit a talk
+
+Send us a message within [Slack](https://links.leidendevs.nl/slack-invite) or [email](mailto:organizers@leiden.dev) to get started. Be sure to include the following information:
+
+* Your talk title
+* Your talk abstract
+* Your talk slides (if applicable)
+* Your talk recording (if applicable)
```

Comment:
You can leave these two points out, a title and a abstract are more than enough.